docs/best-practices/open-source-software-licenties.md

```diff
@@ -79,11 +79,10 @@ oplossing niet te infecteren met de licentie van het hergebruikte component wann
 
 ### Noten
 
-Een OSS licentie uit de smaak ‘Restricted / strong copyleft’ vergt meer kennis van deze betreffende
-OSS licentie om deze juist te kunnen toe te passen. Als je gebruikt maakt van OSS met een OSS
-licentie uit deze smaak dien je met meer voorwaarden rekening te houden. Wanneer je onjuist rekening
-houd met deze voorwaarden loop je het risico dat je aangesproken / aangeklaagd voor
-auteursrechtschending door de oorspronkelijke maker.
+Voor een juiste toepassing van een OSS-licentie uit de smaak ‘Restricted / strong copyleft’ is meer kennis
+nodig. Ze hebben nu eenmaal meer en strengere voorwaarden dan andere licenties. Wanneer hier je onzorgvuldig
+in bent, loop je het risico dat je aangesproken / aangeklaagd wordt voor auteursrechtschending door de
+oorspronkelijke maker.
```

Comment:
Een conventie die ik vaker gezien heb, is om elke zin op een eigen regel te zetten. Dat maakt vergelijken voor een reviewer eenvoudiger, omdat dan alleen de gewijzigde zin wordt aangemerkt als verschil. Misschien ook een idee om hier toe te passen en dan is formattering conventie ook duidelijk?
```suggestion
Voor een juiste toepassing van een OSS-licentie uit de smaak ‘Restricted / strong copyleft’ is meer kennis nodig.
Ze hebben nu eenmaal meer en strengere voorwaarden dan andere licenties.
Wanneer je hier onzorgvuldig in bent, loop je het risico dat je aangesproken / aangeklaagd wordt voor auteursrechtschending door de oorspronkelijke maker.
```
P.s. Er zit in bovenstaande nog wel een correctie: `je` en `hier` omgedraaid.